### ISSUE TITLE
npm run build.prod error

### ISSUE THREAD

**Issue by evilmint** [DeviantJS/angular2-seed-material2#32]
**Steps to reproduce and a minimal demo of the problem**

Execute npm run build.prod

**Current behavior**

```
[14:53:38] 'build.bundles.app' errored after 2.93 s
[14:53:38] Error on fetch for dist\tmp\app\main.js at file:///C:/Users/evilmint/testang/angular2-seed-material2/node_modules/dist\tmp\app\main.js
        Error: ENOENT: no such file or directory, open 'C:\Users\evilmint\testang\angular2-seed-material2\node_modules\dist\tmp\app\main.js'
    at Error (native)
[14:53:38] 'build.prod' errored after 39 s
[14:53:38] Error in plugin 'run-sequence(build.bundles.app)'
Message:
    build.bundles.app callback

npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build.prod"
npm ERR! node v4.4.5
npm ERR! npm  v2.15.5
npm ERR! code ELIFECYCLE
npm ERR! angular2-seed-material2@0.0.0 build.prod: `gulp build.prod --color`
npm ERR! Exit status 1
```

**Expected/desired behavior**

Builds with no problem

**Other information**

It looks like it is looking for dist/ in node_modules/, but I don't know how to fix this.
Happens on a fresh clone of repository, Windows 10

**Comment by d3viant0ne** [comment]
@evilmint - I'll roll this into the updates when angular-material2-alpha.6 is released.

**Comment by d3viant0ne** [comment]
@evilmint - I've fixed this in the advanced-seed material2 fork, I'll have the prod bundles fixed here shortly.

**Comment by d3viant0ne** [comment]
@evilmint - Prod bundles fixed in 83d5e8b82d8f8a609769c5c7e1684b55607ac761
